data/scripts.js

```diff
@@ -699,7 +699,7 @@ let BattleScripts = {
 			}
 		}
 
-		if (move.ohko) this.add('-ohko');
+		if (move.ohko && move.totalDamage === targets[0].hp) this.add('-ohko');
```

Comment:
If we're only trying to detect the case where the foe survives, this would be:

```suggestion
		if (move.ohko && !targets[0].hp) this.add('-ohko');
```

Reply:
If we also don't want to display the message if the foe starts at non-full HP, I would do:

```suggestion
		if (move.ohko && move.totalDamage >= targets[0].hp) this.add('-ohko');
```